api/router.go

```diff
@@ -21,6 +23,7 @@ func Router(c *ctx.AptlyContext) http.Handler {
 	context = c
 
 	router := gin.Default()
+	router.RedirectTrailingSlash = false
```

Comment:
Couldn't this be potentially breaking for some users?

Reply:
Might be, but, unfortunately, without it it does not work for files.. All requests get a trailing slash and `FileFromFS` fails..

Reply:
Maybe this could be configured just for the `/repos` endpoint?
Also it would be nice if this could be enabled/disabled via configuration as some people may not want the files to be served when the api is running?

Reply:
I did my best grepping gin documentation, but it seems to be a global parameter.

Reply:
Hard to say what kind of implications this will have, but the tests seem to pass so at least it doesnt seem to break ...